pwiz_tools/Skyline/CommandArgUsage.resx

```diff
@@ -1052,4 +1052,7 @@ Useful in sample mixtures including multiple species.</value>
   <data name="_associate_proteins_fasta" xml:space="preserve">
     <value>Associate peptides in the document with proteins from the given FASTA. If this argument is not provided but other --associate-proteins options are, then the FASTA from --import-fasta will be used. If --import-fasta is not provided either, then the last FASTA used for protein association will be used.</value>
   </data>
+  <data name="_pep_add_mod_variable" xml:space="preserve">
+    <value>Adds a modification as variable if 'true' or static if 'false'. Only valid for structural modifications with mass.</value>
```

Comment:
Nope. I will try to clarify. This may sound less confusing to users who are not developers, since the UI always shows a delta mass regardless of whether a formula is used. That we don't write that mass to the XML when there is a formula is a low-level developer detail. But I actually mean when mass is non-zero as it is for a loss-only modification.